flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/UserDefinedFunction.scala

```diff
@@ -41,7 +41,7 @@ abstract class UserDefinedFunction extends Serializable {
   def close(): Unit = {}
 
   /**
-    * @return true iff a call to this function is guaranteed to always return
+    * @return true if a call to this function is guaranteed to always return
```

Comment:
I think `iff` is correct. `iff` is means "if and only if".

Reply:
We should be writing out "if and only if" iff that is the intent.

Reply:
thanks all, I think "if and only if" is better